### DISCUSSION TITLE
How to get rid of borders / apply app theme

### DISCUSSION THREAD

**Discussion by redarflap** [danvick/flutter_form_builder#1111] [Q&A] [answered]
Hi, many of the fields seem to come with an enormous input decoration border.

I'm looking for a much more minimalistic design. How could i get rid of those borders?
In fact I'd like everything to look like defined in my app's theme.

Passing the following doesn't seem to get rid of it:

```
decoration: const InputDecoration(
                  contentPadding: EdgeInsets.only(top: 0.0, bottom: 0.0),
                  border: InputBorder.none,
                ),
```

See Form Builder Slider / Dropdown vs generic Slider/Dropdown
![grafik](https://user-images.githubusercontent.com/85738322/189650127-f42678b0-0c73-461c-8e20-0ab26b6bb0a2.png)

**Comment by deandreamatias** [comment, marked as answer]
Hi!
The class InputDecoration has more properties than these, like enableBorder. You can try configure this properties.
If don't work, you can create your own widget too, take a look on `form_builder_dropdown.dart` file
